www/app/chats/chats.controller.js

```diff
@@ -0,0 +1,10 @@
+ (function() {
```

Comment:
what is this controller for?
i saw chatpage before and now chats? 
what is difference